units/CHANGELOG.md

```diff
@@ -1,8 +1,31 @@
-# Unreleased
-
-- TODO: Make a comment about `Amount::MAX_MONEY` including breaking serde
-
-- Use MAX_MONEY in serde regression test [#3950](https://github.com/rust-bitcoin/rust-bitcoin/pull/3950)
+# 1.0.0-alpha - 2025-02-24
+
+Release everything except the `amount` types.
+
+BOOM! A long time in the making but here goes, our first alpha 1.0 crate release.
+
+* Add `FIFTY_BTC` const to the amount types [#3915](https://github.com/rust-bitcoin/rust-bitcoin/pull/3915)
+* Remove `InputString` from the public API [#3905](https://github.com/rust-bitcoin/rust-bitcoin/pull/)
+* Hide the remaining public macros [#3867]()
+* Introduce an unchecked constructor for the `Amount` type [#3811]()
```

Comment:
We agreed to remove it though.

Reply:
Ugh, I didn't even re-read any of these changelog entries when re-doing the PR - fail.